source/sessions/tests/unified/snapshot-sessions.json

```diff
@@ -0,0 +1,698 @@
+{
+  "description": "snapshot-sessions",
+  "schemaVersion": "1.0",
+  "runOnRequirements": [
+    {
+      "minServerVersion": "5.0",
+      "topologies": [
+        "sharded-replicaset"
+      ]
+    }
+  ],
+  "createEntities": [
+    {
+      "client": {
+        "id": "client0",
+        "observeEvents": [
+          "commandStartedEvent"
+        ]
+      }
+    },
+    {
+      "database": {
+        "id": "database0",
+        "client": "client0",
+        "databaseName": "database0"
+      }
+    },
+    {
+      "collection": {
+        "id": "collection0",
+        "database": "database0",
+        "collectionName": "collection0",
+        "collectionOptions": {
+          "readPreference": {
+            "mode": "secondary"
+          }
+        }
+      }
+    },
+    {
+      "session": {
```

Comment:
Is there really a need for all these session objects?  Each test runs in isolation, so a new session instance is created for each test.  So there is no re-use issues between tests.  I suspect we can have just one session with an isSnapshot: true option.

Reply:
No need, removed.